tcms/issuetracker/tests/test_bitbucket.py

```diff
@@ -0,0 +1,188 @@
+# pylint: disable=attribute-defined-outside-init
+import os
+import unittest
+
+from django.utils import timezone
+
+from tcms.core.contrib.linkreference.models import LinkReference
+from tcms.issuetracker.bitbucket import BitBucketAPI, BitBucket
+from tcms.rpc.tests.utils import APITestCase
+from tcms.testcases.models import BugSystem
+
+from tcms.tests.factories import ComponentFactory
+from tcms.tests.factories import TestExecutionFactory
+
+
+# TODO: BITBUCKET_INTEGRATION_API_PASSWORD = BitBucket app password
+# TODO: BITBUCKET_INTEGRATION_API_USERNAME = BitBucket username
+@unittest.skipUnless(os.getenv('TEST_BUGTRACKER_INTEGRATION'),
+                     'Bug tracker integration testing not enabled')
+class TestBitBucketIntegration(APITestCase):
+    existing_bug_id = 1
+    # TODO: Will be replaced with issue for test in Kiwi CI.
+    existing_bug_url = 'https://bitbucket.org/selftronics/sample/issues/1'
```

Comment:
I've created a new repository and an issue:
https://bitbucket.org/kiwitcms/integration/issues/1/hello-world

which is supposed to be the the internal BitBucket Issue tracker, not their JIRA integration.